packages/web-components/src/components/image-with-caption/__tests__/image-with-caption.test.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { render } from 'lit-html';
+import '../image-with-caption';
+import { Default } from '../__stories__/image-with-caption.stories.ts';
```

Comment:
To avoid the TS error:

```suggestion
import { Default } from '../__stories__/image-with-caption.stories';
```

Reply:
fixed! @asudoh